src/schema/schema_worker.c

```diff
@@ -124,7 +130,11 @@ __wt_schema_worker(WT_SESSION_IMPL *session, const char *uri,
          */
         for (i = 0; i < WT_COLGROUPS(table); i++) {
             colgroup = table->cgroups[i];
-            skip = false;
+            /*
+             * FIXME-WT-10520 - Don't skip verifying tiered tables once it is supported. (Change the
+             * statement below to skip = false.)
+             */
+            skip = (file_func == __wt_verify) && WT_PREFIX_MATCH(colgroup->source, "tiered:");
```

Comment:
I think we should return ENOTSUP here if we have a tiered table. 

In theory we could have a complex column group that is a mix of tiered tables and other tables and we wouldn't verify those other tables. but since tiered tables aren't supported yet, I don't think we need to worry about that, and it would be better to tell the user (via ENOTSUP) that we didn't verify the tiered table.